plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -112,7 +112,7 @@ export interface RouterOptions {
   taskBroker?: TaskBroker;
   additionalTemplateFilters?: Record<string, TemplateFilter>;
   additionalTemplateGlobals?: Record<string, TemplateGlobal>;
-  permissionApi?: PermissionEvaluator;
```

Comment:
hmm this would be a breaking change and not something we could release as a `minor` version. We might have to keep calling this one permissionApi to not introduce a new major version just jet. Alternatively deprecate this one and introduce `permission`.

Reply:
`permissionApi` was introduced in one of the previous next releases and hasn't been released in a final version yet, so we should be able to change it, right? 🤔

Reply:
If that's the case and it's not in the previous mainline release then es. But I was under the impression that the older permission stuff already got released no?

Reply:
If you can confirm that it's not in the v1.12 release then go ahead and ship this, I will have limited access tomorrow but would be good to get this in the release :pray:

Reply:
yes, it was introduced in `1.13.0-next.1` 😄
https://backstage.github.io/upgrade-helper/?from=1.12.0&to=1.13.0-next.1

Reply:
Nice! Thanks for checking @vinzscam, in that case this is good to go 👍